src/main/java/com/spotify/docker/client/messages/ServiceCreateResponse.java

```diff
@@ -36,8 +40,21 @@
   @JsonProperty("ID")
   public abstract String id();
 
+  @Nullable
+  @JsonProperty("Warnings")
+  public abstract ImmutableList<String> warnings();
+
   @JsonCreator
-  static ServiceCreateResponse create(@JsonProperty("ID") final String id) {
-    return new AutoValue_ServiceCreateResponse(id);
+  static ServiceCreateResponse create(
+          @JsonProperty("ID") final String id) {
+    return create(id, null);
+  }
+  
+  static ServiceCreateResponse create(
```

Comment:
Same comment as in [798](https://github.com/spotify/docker-client/pull/798#discussion_r122107311). This `create` needs to be the `@JsonCreator`, or `"Warnings"` would never be deserialized from json.